tests/api/test_nanopub_crud.py

```diff
@@ -29,7 +29,6 @@ def test_read(self):
         response = self.post_nanopub(data=self.turtle,
                                         content_type="text/turtle",
                                         expected_headers=["Location"])
-
```

Comment:
Nit: don't change files unrelated to what you're doing, because it shows up in the history. Fine to leave this this time.